src/main/java/frc/robot/auto/AutoPaths.java

```diff
@@ -0,0 +1,73 @@
+package frc.robot.auto;
+
+import java.lang.reflect.Field;
+import java.util.HashMap;
+
+import frc.robot.constants.AutoConstants.AutoCommands;
+
+public class AutoPaths {
+	public static final Object[] B_PATH_1 = new Object[] {
+		new Object[] {
```

Comment:
Is there any way we could put all the paths into the AutoCommands enum? This mixed type stuff is really messy and error-prone.